src/main/java/com/sparta/popupstore/domain/review/controller/ReviewController.java

```diff
@@ -59,4 +66,10 @@ public ResponseEntity<Void> deleteReview(@AuthUser User user ,
         reviewService.deleteReview(user, reviewId);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+
+    @GetMapping("/popupstores/{popupStoreId}")
+    public Page<ReviewFindAllResponseDto> findReviews(@AuthUser User user, @PathVariable Long popupStoreId,
+        @PageableDefault(size = 10, sort = "updatedAt", direction = Sort.Direction.DESC) Pageable pageable) {
```

Comment:
```suggestion
        @PageableDefault(sort = "updatedAt", direction = Sort.Direction.DESC) Pageable pageable) {
```
size의 기본 default 값이 10 입니다.